tools/export/gnuarmeclipse/__init__.py

```diff
@@ -1014,8 +1014,7 @@ def process_options(self, opts, flags_in):
         opts['cpp']['other'] += ' ' + \
             ' '.join(flags['common_flags']) + ' ' + \
             ' '.join(flags['cxx_flags'])
-        opts['ld']['other'] += ' ' + \
-            ' '.join(flags['common_flags']) + ' ' + ' '.join(flags['ld_flags'])
+        opts['ld']['other'] += ' ' +  ' '.join(flags['ld_flags'])
```

Comment:
That's not how the mbed-os tools do it. I take it that this is also non-standard for eclipse.